docs/multi-stage-query/api.md

```diff
@@ -528,6 +530,10 @@ The response shows an example report for a query.
               "frames": [
                 73
               ]
+            },
+            "segmentGenerationProgress": {
+              "type": "segmentGenerationProgress",
```

Comment:
Should the counter should have the key "output" and then output type is "segmentGenerationProgress"
Does that interfere with the current metrics ?

Reply:
Yes, I think it would cause a problem since we already use that name. I think ideally we should keep the names unique, to avoid any exceptions, since we cast to the child class after fetching based on channel name. 
If we want "output" in the name to make it clear, we could rename this to "segmentGenerationOutput"?

Reply:
@cryptoe I do not understand your suggestion. Could you please provide an example of a JSON so help illustrate your point?

The current structure seems good to me although maybe I am missing something.